openmetadata-spec/src/main/resources/json/schema/entity/services/connections/storage/s3Connection.json

```diff
@@ -24,6 +24,12 @@
       "title": "AWS Credentials Configuration",
       "$ref": "../../../../security/credentials/awsCredentials.json"
     },
+    "bucketName": {
```

Comment:
Would it make sense to support `n` number of buckets?

Reply:
technically that is supported if you leave this field empty and put filters this is what we follow even for databases right 🤔

Reply:
I see; what would be the difference with setting a bucketName vs adding a bucket name to the filter?

Reply:
so when you want to ingest multiple buckets you need `ListBuckets` permission, this pr is to cover the scenario when user do not want to give permission for `ListBuckets` due to some security reasons and only wants to ingest a single bucket.


so when using a filter u need `ListBuckets` permissions, when using `bucketName` you do not need that permission